storage/storage_tests/storage_tests.cpp

```diff
@@ -1227,4 +1221,39 @@ UNIT_TEST(StorageTest_ParseStatus)
   TEST_EQUAL(StatusAndError(NodeStatus::Downloading, NodeErrorCode::NoError),
              ParseStatus(Status::EDownloading), ());
 }
+
+UNIT_TEST(StorageTest_ForEachInSubtree)
+{
+  Storage storage(kSingleMwmCountriesTxt, make_unique<TestMapFilesDownloader>());
+
+  TCountriesVec leafVec;
+  auto const forEach = [&leafVec](TCountryId const & descendantCountryId, bool expandableNode)
+  {
+    if (!expandableNode)
+      leafVec.push_back(descendantCountryId);
+  };
+  storage.ForEachInSubtree(storage.GetRootId(), forEach);
+
+  TCountriesVec const expectedLeafVec = {"Abkhazia", "Algeria_Central", "Algeria_Coast", "South Korea_South"};
+  TEST_EQUAL(leafVec, expectedLeafVec, ());
+}
+
+UNIT_TEST(StorageTest_CalcLimitRect)
+{
+  Storage storage(COUNTRIES_FILE);
+  if (!version::IsSingleMwm(storage.GetCurrentDataVersion()))
```

Comment:
The only difference I see is that the one tests "Angola" and the other tests "Algeria".